doc/ROADMAP.rst.txt

```diff
@@ -259,7 +259,13 @@ things are done in `interpolate`), and eliminate any duplication.
 *Continuous-Time Linear Systems*: remove `lsim2`, `impulse2`, `step2`.  Make
 `lsim`, `impulse` and `step` "just work" for any input system.  Improve
 performance of ltisys (less internal transformations between different
-representations).
+representations). Fill gaps in lti system conversion functions.
+
+*Second Order Sections*: Make SOS filtering equally capable as existing
+methods. This includes ltisys objects, `freqz` and `lfiltic` equivalents, and
```

Comment:
`lfiltic` is the generalization of `lfilter_zi`, so it would be nice to have the equivalent of `sosfilt_ic` to generalize `sosfilt_zi`. In the recent past, I wanted this function, though I can't remember exactly what for right now...

Good point about `sosfreqz`, though. This was basically copy-and-pasted from some notes to myself from a while back...